x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -1316,6 +1504,24 @@ var security = (function () {
         })
         .Build();
 
+    var addGroupType = function(evt) {
+        var code = evt.Get("event.code");
+        if (!code) {
+         return;
+        }
+        evt.Put("winlog.group.type", group_types[code][0]);
```

Comment:
Getting an error here for events 4764 and 4799, looks like those events don't have an entry in group_types.  We should probably add those and we might want to add a check to make sure there are entries before we do the evt.Put